lib/tjbot.js

```diff
@@ -659,6 +659,9 @@ TJBot.prototype.listen = function(callback) {
         if (err) {
             winston.error("the speech_to_text service returned an error.", err);
 
+            //stop tje microphone
```

Comment:
"tje"?  ;)

note that the call to `self.resumeListening()` is unnecessary at this point as `stopListening()` will kill the underlying `arecord` process, so `resumeListening()` won't actually do anything (there is no more process to resume).  it can safely be removed.

thanks!